src/client/components/FavouriteList/ProductQuantity.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import './FavouriteList.styles.css';
+
+const ProductQuantity = ({ product }) => {
+  const [counter, setCounter] = useState(0);
```

Comment:
I suggest starting the counter at 1, not 0. Because we cannot add 0 qty to the cart, agree?